interviews/backend-takehome.md

```diff
@@ -38,3 +38,16 @@ Note that bonus points may be reduced if Solution Quality is not great.
 - ⭐️ 4: Requested delighter content completed fully.
 - 👍 3: Requested delighter mostly completed or completed with a few minor bugs/flaws.
 - 👎 2: Little to no delighter was completed.
+
+## What We Look For
```

Comment:
Likewise, maybe add a link to a  `## Backend Specifics` section to `### Backend Knowledge`, similar to how frontend one is set up